### PR TITLE
fix: create preview for draft PRs

### PR DESCRIPTION
**Pull request by prescottprue**
:tickets: TICKET-000 

## Changes
Create preview url for draft PRs while still skipping Percy for draft and renovate PRs - this was per request: https://top-agent.slack.com/archives/C02S7T34ZHR/p1724858084769979

### CODE REVIEW

File: .github/actions/storybook/action.yml [hunk 1]
```diff
@@ -4,6 +4,9 @@ inputs:
   percy-token:
     description: 'Token for Percy'
     required: true
+  skip-percy:
+    description: 'Whether or not to skip Percy - if skipped build still runs and preview url is still generated'
+    required: false
```
Comment (skydiator) [inline .github/actions/storybook/action.yml:9]: should we use default here?

**Review by skydiator — CHANGES_REQUESTED** [review]

File: .github/actions/storybook/action.yml (hunk 1 above)
Comment (prescottprue) [reply .github/actions/storybook/action.yml:9]: I just check for `== 'true'` below so it isn't needed, but yeah I can add

**Comment by prescottprue** [comment]
@skydiator Added the default and switched back to v1 - confirmed functionality here: https://github.com/reside-eng/pantry/pull/2118, so this should be good to go now

File: .github/actions/storybook/action.yml (hunk 1 above)
Comment (skydiator) [reply .github/actions/storybook/action.yml:9]: Thanks for adding that. All LGTM.

**Review by skydiator — APPROVED** [review]

**Pull request merged**